charts/vault/Makefile

```diff
@@ -0,0 +1,32 @@
+SHELL := /bin/bash # Use bash syntax
+ROOT_DIR:=../..
+include $(ROOT_DIR)/Makefile.env
+include $(ROOT_DIR)/hack/make-rules/docker.mk
+
+KUBE_NAMESPACE ?= m4d-system
+WITHOUT_OPENSHIFT ?= true
+
+.PHONY: install
+install:
+	kubectl create namespace $(KUBE_NAMESPACE) 2>/dev/null || true
+	helm repo add hashicorp https://helm.releases.hashicorp.com
+        ifeq ($(WITHOUT_OPENSHIFT), true)
+	$(info installing Vault without openshift)
+	helm install vault hashicorp/vault --values values/plugin-secrets-kubernetes-reader.yaml
+        else
+	helm install vault hashicorp/vault --values values/plugin-secrets-kubernetes-reader.yaml --values values/openshift.yaml
+        endif
+	kubectl create secret generic vault-unseal-keys --from-literal=vault-root=root -n $(KUBE_NAMESPACE) || true
```

Comment:
What's this secret?

Reply:
Its the vault token. its been used in the project to authenticate to vault (in the manager for example)... the installation in third_party/vault seems to create it automatically, the following is from the vault-0 pod created with banzaicloud repo:
  - args:
    - --pre-flight-checks
    - "true"
    - --mode
    - k8s
    - --k8s-secret-namespace
    - m4d-system
    - --k8s-secret-name
    - vault-unseal-keys
    - --k8s-secret-labels
    - app.kubernetes.io/name=vault,vault_cr=vault

Reply:
Okay, let's talk about that for a follow-up, doesn't seem right. Unsealing is a separate process (not required in dev mode btw). Login should use a proper method even from the manager.